Mathlib/LinearAlgebra/QuadraticForm/TensorProduct.lean

```diff
@@ -95,6 +95,13 @@ protected noncomputable abbrev tmul (Q₁ : QuadraticForm A M₁) (Q₂ : Quadra
     QuadraticForm A (M₁ ⊗[R] M₂) :=
   tensorDistrib R A (Q₁ ⊗ₜ[R] Q₂)
 
+/- Previously `QuadraticForm.tensorDistrib` was defined in a similar way to
+`QuadraticMap.tensorDistrib`. We now obtain the definition of `QuadraticForm.tensorDistrib`
+from `QuadraticMap.tensorDistrib` using `A ⊗[R] R ≃ₗ[A] A`. Hypothesis `e1` below shows that the
+new definition is equal to the old, and allows us to reuse the old proof.
+
+TODO: Define `IsSymm` for bilinear maps and generalise this result to Quadratic Maps.
+-/
```

Comment:
Can you put this inside the proof just before `have e1`? After all, this is a comment about the proof, not the theorem

Reply:
Done, thanks.

Reply:
For a proper solution we probably want something like:
```
variable [AddCommMonoid M] [CommSemiring R] [Module R M] [StarRing R] [Star M]

/-- The proposition that a sesquilinear form is conjugate symmetric -/
def LinearMap.IsSymm (B : M →ₛₗ[starRingEnd R] M →ₗ[R] R) : Prop :=
  ∀ x y, star (B x y) = B y x
```

But preferably with `[NonUnitalSemiring R]` (or even `[NonUnitalNonAssocSemiringSemiring R]`) instead of `[CommSemiring R]` in order that this would cover Hilbert C*-modules.

Reply:
I've come up with this: #18406